lib/atecc508a/transport/i2c_server.ex

```diff
@@ -158,14 +158,8 @@ defmodule ATECC508A.Transport.I2CServer do
   defp make_cached_request(payload, timeout, response_payload_len, i2c, address, cache) do
     case Cache.get(cache, payload) do
       nil ->
-        case make_request(payload, timeout, response_payload_len, i2c, address) do
-          {:ok, _message} = rc ->
-            Cache.put(cache, payload, rc)
-            rc
-
-          error ->
-            error
-        end
+        result = make_request(payload, timeout, response_payload_len, i2c, address)
+        Cache.put(cache, payload, result)
```

Comment:
```suggestion
        Cache.put(cache, payload, result)
        result
```

`Cache.put` only returns `:ok` which breaks the spec here so result needs to be returned after

Reply:
This isn't needed, but it actually looks more clear to me.

Reply:
O..yep...there it is 🤦🏼